.codeclimate.yml

```diff
@@ -30,15 +30,20 @@ engines:
     enabled: true
   fixme:
     enabled: true
+    checks:
+      XXX:
+        enabled: false
   rubocop:
     enabled: true
     checks:
       Rubocop/Style/SingleLineBlockParams:
         enabled: false
-      Rubocop/Rails/DynamicFindBy:
-        enabled: false
       Rubocop/Style/AndOr:
         enabled: false
+      Rubocop/Style/IfUnlessModifier:
+        enabled: false
```

Comment:
This cop enforces that if `if` and `unless` blocks are only a single line the conditional should be moved onto the same line instead of using a block.  However, this often causes the resulting line to be unreasonable long.  If I had to choose between the two, I would choose manageable and reasonable lines so I'm disabling this cop.